ambry-store/src/main/java/com/github/ambry/store/StorageManager.java

```diff
@@ -165,8 +165,8 @@ private void verifyConfigs(StoreConfig storeConfig, DiskManagerConfig diskManage
     /* NOTE: We must ensure that the store never performs hard deletes on the part of the log that is not yet flushed.
        We do this by making sure that the retention period for deleted messages (which determines the end point for hard
        deletes) is always greater than the log flush period. */
-    if (storeConfig.storeDeletedMessageRetentionDays
-        < TimeUnit.SECONDS.toDays(storeConfig.storeDataFlushIntervalSeconds) + 1) {
+    if (storeConfig.storeDeletedMessageRetentionHours
+        < TimeUnit.SECONDS.toHours(storeConfig.storeDataFlushIntervalSeconds) + 1) {
       throw new StoreException("Message retention days must be greater than the store flush interval period",
```

Comment:
minor: Might want to change to "Message retention hours" for the exception message.